java/java/JavaLexer.g4

```diff
@@ -97,6 +99,17 @@ TRANSITIVE:         'transitive';
 // Local Variable Type Inference
 VAR:                'var'; // reserved type name
 
+// Switch Expressions
+YIELD:              'yield';
+
+// Records
+RECORD:             'record';
+
+// Sealed Classes
+SEALED:             'sealed';
+PERMITS:            'permits';
```

Comment:
They all also can be identifiers. Please all `yield` `record`, `sealed`, `permits` to `identifier` rule.

Reply:
> They all also can be identifiers. Please all `yield` `record`, `sealed`, `permits` to `identifier` rule.

it's already there

Reply:
Sorry, I missed that. But `yield` is still missing and it's an acceptable identifier as well.